apps/charterafrica/src/payload/collections/Tools.js

```diff
@@ -0,0 +1,217 @@
+import {
+  TOOL_COLLECTION,
+  ORGANIZATION_COLLECTION,
+  PEOPLE_COLLECTION,
+} from "../../lib/tools/models";
+import deletedAt from "../fields/deletedAt";
+import slug from "../fields/slug";
+import source from "../fields/toolSourceField";
+import updatedAt from "../fields/updatedAt";
+
+const Tools = {
+  slug: TOOL_COLLECTION,
+  admin: {
+    useAsTitle: "externalId",
+  },
+  access: {
+    read: () => true,
+  },
+  labels: {
+    singular: {
+      en: "Tool",
+      fr: "Outil",
+      pt: "Ferramenta",
+    },
+    plural: {
+      en: "Tools",
+      fr: "Outils",
+      pt: "Ferramentas",
+    },
+  },
+  fields: [
+    source(),
+    {
+      name: "externalId",
+      type: "text",
+      required: true,
+      label: { en: "External ID", fr: "ID externe", pt: "ID externo" },
+      admin: {
+        readOnly: true,
+      },
+    },
+    slug({ fieldToUse: "name" }),
+    {
+      name: "name",
+      type: "text",
+      label: {
+        en: "Tool Name",
+        fr: "Nom de l'outil",
+        pt: "Nome da ferramenta",
+      },
+      admin: {
+        readOnly: true,
+      },
+    },
+    {
+      name: "description",
+      type: "textarea",
+      admin: {
+        readOnly: true,
+      },
+      label: {
+        en: "Tool Description",
+        fr: "Description de l'outil",
+        pt: "Descrição da ferramenta",
+      },
+    },
+    {
+      name: "link",
+      type: "text",
+      admin: {
+        readOnly: true,
+      },
+      label: {
+        en: "Tool Link",
+        fr: "Lien de l'outil",
+        pt: "Link da ferramenta",
+      },
+    },
+    {
+      name: "location",
+      type: "text",
+      admin: {
+        readOnly: true,
+      },
+      label: {
+        en: "Tool Location",
+        fr: "Emplacement de l'outil",
+        pt: "Localização da ferramenta",
+      },
+    },
+    {
+      name: "subject",
+      type: "text",
+      admin: {
+        readOnly: true,
+      },
+      label: { en: "Subject", fr: "Sujet", pt: "Assunto" },
+    },
+    {
+      name: "languagesTechSkills",
```

Comment:
Lets go with a simple name: technologies, techStack, etc. They cover both languages and other skills I believe.